make

```diff
@@ -78,6 +87,22 @@ function main() {
     todo)
       grep -E 'TODO:|FIXME:' **/*.[ch]
       ;;
+    docker-build)
+      shift
+      # Cygwin is its own special case; Linux/Mac/WSL can be handled the same way
+      if [[ $(uname -a | grep -i cygwin >/dev/null) ]]
+      then
+        SCRIPTPATH_CYGWIN="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+        if [[ "${SCRIPTPATH_CYGWIN}" =~ "^/cygdrive" ]]; then
+          SRC_PATH="$(sed 's|cygdrive||' <<< \"${SCRIPTPATH_CYGWIN}\")" # e.g. /cygdrive/c/users/foo -> //c/users/foo
+        else
+          SRC_PATH="$(cygpath -m -w /)/${SCRIPTPATH_CYGWIN}"
+        fi
+      else
+        SRC_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" # Handles various edge cases better than a naive ${PWD} or $(pwd)
+      fi
+      docker run --pull=true --rm -v ${SRC_PATH}:/src -it siterelenby/anduril-builder:latest ${@}
```

Comment:
Why not just include the docker file?

Reply:
The actual docker image build is very slow and not something the average user wants or needs to do; I make multiarch images available on Docker Hub to avoid that. The repo itself could be included as a submodule as well, I guess, but this way, this allows anyone with a working Docker* (no matter their OS platform) to build anduril without needing to worry about dependencies or spend the time to build (and have to maintain updating) their own Docker builder.

* Well, I should probably add a .bat for Windows users without WSL, but I don't feel like doing that today, heh...

Reply:
I guess I could add something like build-local-docker-image as well in case someone really wants to build their own, but at that point might as well just clone the repo/submodule themselves to do it.